docs/wiki/testing/incidents.md

```diff
@@ -1,6 +1,12 @@
-# Notable mainnet incidents 
+# Notable mainnet incidents
 
 > :warning: This article is a [stub](https://en.wikipedia.org/wiki/Wikipedia:Stub), help the wiki by [contributing](/contributing.md) and expanding it.
+> Incidents are the disruptions, vulnerabilities and attacks faced by the network which has questioned it's stability and security. Below are some incidents that affected the network and their sources.
 
-- [Post-Mortem Report: Ethereum Mainnet Finality (05/11/2023)](https://medium.com/offchainlabs/post-mortem-report-ethereum-mainnet-finality-05-11-2023-95e271dfd8b2)
-- [Minority split 2021-08-27 post mortem](https://github.com/ethereum/go-ethereum/blob/master/docs/postmortems/2021-08-22-split-postmortem.md)
\ No newline at end of file
+- [Post-Mortem Report: Ethereum Mainnet Missing Blocks (27/03/2024)](https://u.today/ethereum-developers-investigate-incident-that-triggered-missing-blocks-on-eth-mainnet) Earlier in March 2024, the beacon chain faced a disruption which characterized in some blocks get missing from the network, this incident went on for about a week before it was resolved by the core engineers, sources are yet to discover if this was triggered by the dencun upgrade.
```

Comment:
The headline here sounds a bit weird. Instead I would link the technical writeup and discussion about it 
```suggestion
- [Post-Mortem, Blob Propagation Issues (27/03/2024)](https://gist.github.com/benhenryhunter/687299bcfe064674537dc9348d771e83) In March 2024, after Dencun upgrade, blobs attached to blocks from certain builders propagated too slowly over p2p which caused a client implementation to miss few slots. 
```